qiskit/ignis/verification/topological_codes/circuits.py

```diff
@@ -197,33 +194,204 @@ def process_results(self, raw_results):
             for string in raw_results[log]:
 
                 # logical readout taken from
-                measured_log = string[0] + ' ' + string[self.d - 1]
+                measured_log = string[0] + " " + string[self.d - 1]
 
                 # final syndrome deduced from final code qubit readout
-                full_syndrome = ''
+                full_syndrome = ""
                 for j in range(self.d - 1):
-                    full_syndrome += '0' * (string[j] == string[j + 1]) \
-                        + '1' * (string[j] != string[j + 1])
+                    full_syndrome += "0" * (string[j] == string[j + 1]) + "1" * (
+                        string[j] != string[j + 1]
+                    )
                 # results from all other syndrome measurements then added
                 full_syndrome = full_syndrome + string[self.d:]
 
                 # changes between one syndrome and the next then calculated
-                syndrome_list = full_syndrome.split(' ')
-                syndrome_changes = ''
+                syndrome_list = full_syndrome.split(" ")
+                syndrome_changes = ""
                 for t in range(self.T + 1):
                     for j in range(self.d - 1):
                         if t == 0:
-                            change = (syndrome_list[-1][j] != '0')
+                            change = syndrome_list[-1][j] != "0"
                         else:
-                            change = (syndrome_list[-t][j]
-                                      != syndrome_list[-t - 1][j])
-                        syndrome_changes += '0' * (not change) + '1' * change
-                    syndrome_changes += ' '
+                            change = syndrome_list[-t][j] != syndrome_list[-t - 1][j]
+                        syndrome_changes += "0" * (not change) + "1" * change
+                    syndrome_changes += " "
 
                 # the space separated string of syndrome changes then gets a
                 # double space separated logical value on the end
-                new_string = measured_log + '  ' + syndrome_changes[:-1]
+                new_string = measured_log + "  " + syndrome_changes[:-1]
 
                 results[log][new_string] = raw_results[log][string]
 
         return results
+
+    def _get_all_processed_results(self):
+        """
+        Returns:
+            results: list of all processed results stemming from single qubit bitflip errors,
+                which is used to create the decoder graph.
+        """
+
+        syn = RepetitionCodeSyndromeGenerator(self)
+
+        T = syn.T  # number of rounds of stabilizer measurements
+        d = syn.d  # number of data qubits
```

Comment:
This feels a bit weird to me because I read this as `T = syn.T = self.T` and nothing modifies it. Wouldn't it just be easier to use `self.T`?

Reply:
I think it's mostly for consistency with the other `syn.` things done in the following block. So to show that these are the corresponding values of `T` and `d`. But yes, it should be equivalent.

Reply:
I think it's fine to leave as is